modules/wizard/wizard.php

```diff
@@ -652,14 +653,14 @@ public function save_step_media() {
 	 */
 	public function add_step_untranslated_contents( $steps ) {
 		if ( ! $this->model->get_languages_list() || $this->model->get_objects_with_no_lang( 1 ) ) {
-			wp_enqueue_script( 'pll_admin', plugins_url( '/js/admin' . $this->get_suffix() . '.js', POLYLANG_FILE ), array( 'jquery', 'wp-ajax-response', 'postbox', 'jquery-ui-selectmenu' ), POLYLANG_VERSION, true );
+			wp_enqueue_script( 'pll_admin', plugins_url( '/js/admin' . $this->get_suffix() . '.js', POLYLANG_FILE ), array( 'jquery', 'jquery-ui-selectmenu' ), POLYLANG_VERSION, true );
```

Comment:
It seems this script will never be enqueued because a script with the same handle will always be enqueued at line 467.

Reply:
Exactly! It isn't necessary because we are sure the Languages step does the job just before.
However we have to see each step independent of the others because one step could be not added to the wizard.